.github/workflows/release.yaml

```diff
@@ -0,0 +1,338 @@
+name: release
+
+# TODO - the real trigger
+# on:
+#   push:
+#     tags:
+#       - 'v*'
+
+# FAKE for testing
+on:
+  pull_request:
+    paths:
+      - '**/*'
+      - '!docs/**'
+      - '!examples/**'
+      - '!README.md'
+
+
+# TODO - wire up signing for official releasese
+
+jobs:
+  # setup the version for easy consumption in the other jobs
+  version:
+    runs-on: ubuntu-latest
+    steps:
+    # - uses: 'google-github-actions/auth@v2'
+    #   with:
+    #     project_id: 'ollama'
+    #     workload_identity_provider: '${{ secret goes here }}'
+    # - run: |
+    #     gcloud projects list
+    - name: Get the target release version
+      id: get_version
+      run: |
+        ver=${GITHUB_REF/refs\/tags\//}
+        echo "VERSION=0.0.0" >> "$GITHUB_OUTPUT"
+      #  echo "VERSION::${ver#v}" >> "$GITHUB_OUTPUT"
+    outputs:
+      VERSION: ${{ steps.get_version.outputs.VERSION }}
+
+  # Full build of the Mac assets
+  build-darwin:
+    needs:
+      - version
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          cache: true
+      - run: |
+          export VERSION="${{ needs.version.outputs.VERSION }}"
+          ./scripts/build_darwin.sh
+      - uses: actions/upload-artifact@v4
+        with:
+          name: dist-darwin
+          path: |
+            dist/*arwin*

```

Comment:
is the directory not deterministic?

Reply:
I thought at one point I saw a file with Darwin (cap D) so I was making sure it would pick that up too, but double checking, it's the Ollama vs ollama that's inconsistent in case between the zip file and unified binary.  Basically my intent here was to grab everything darwin releated.